modules/ROOT/pages/7.7.0-release-notes.adoc

```diff
@@ -141,6 +141,13 @@ the help dialog would not open if the plugin was disabled, leading to confusion.
 
 {productname} {release-version} addresses this issue by ensuring that if the help plugin is disabled, the screen reader announces only "Rich Text Area." If the help plugin is enabled, the screen reader announces "Rich Text Area. Press `ALT-0` for help.".
 
+=== Toolbar groups had both a `title` attribute and a custom tooltip, causing overlapping tooltips
+// #TINY-11768
+
+In previous versions of {productname}, hovering over toolbar entries would display both a custom tooltip and the default browser tooltip. This caused confusing behavior, as the custom tooltip was difficult to read due to the browser tooltip overlapping it.
```

Comment:
I'm not sure how detailed you want to be with this. The custom tooltip was the name of the button, while the browser's default tooltip had the name of the button group that the user was over. Fully correct as-is though.